plenoptic/simulate/canonical_computations/pooling_windows.py

```diff
@@ -15,6 +15,7 @@
 import matplotlib.pyplot as plt
 import os.path as op
 from ...tools.data import to_numpy
+from ...tools.display import imshow
```

Comment:
There are a fair number of relative imports in our package now.  I have found their presence to be somewhat confusing on a number of occasions.  PEP8 seems to discourage relative imports to some degree (https://www.python.org/dev/peps/pep-0008/#imports).  Thoughts?

Reply:
Oh woah, my impression was that absolute imports from within a package were a bad idea, because it is possible to have multiple versions of the same package on your path (for example, if you have a local version of a package and then are in a virtual environment where you've installed it via `pip`), and I thought the relative import would make sure it's grabbing the version from within your package, rather than some other version on the path. We should look into this more.

Reply:
In fact I just told Lyndon to change an absolute to relative import in #73

Reply:
I think I might be using the wrong language here... basically I'm concerned by the fact that that we import a bunch of variables and method names and then when we call them, it's not immediately obvious that they come from outside the current folder.  So instead of calling clean_up_axes(), could we call tools.display.clean_up_axes()

Reply:
I absolutely agree with the change you recommended in #73 but I think I'd like to ask that we still use the full path to call the methods.

Reply:
Oh gotcha. That makes a lot of sense.

So the top would say somethign like `import ...tools` instead